domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Sats.kt

```diff
@@ -1,13 +1,36 @@
 package no.nav.su.se.bakover.domain.beregning
 
+import no.nav.su.se.bakover.common.periode.Periode
 import no.nav.su.se.bakover.domain.Grunnbeløp
 import java.math.BigDecimal
 import java.time.LocalDate
+import kotlin.math.roundToInt
 
 enum class Sats(val grunnbeløp: Grunnbeløp) {
     ORDINÆR(Grunnbeløp.`2,28G`),
     HØY(Grunnbeløp.`2,48G`);
 
-    fun fraDato(dato: LocalDate): Double = grunnbeløp.fraDato(dato)
-    fun fraDatoAsInt(dato: LocalDate): Int = BigDecimal.valueOf(grunnbeløp.fraDato(dato)).divide(BigDecimal.ONE, 0, java.math.RoundingMode.HALF_UP).toInt()
+    private val MÅNEDER = 12L
+    private val MÅNEDER_PRESISJON = 2
```

Comment:
Hva er greia med presisjon her og for år? Hva gjør det av vi spesifiserer det?

Reply:
Antall desimaler.